config/defaults.conf

```diff
@@ -435,6 +435,13 @@ glue-small-tasks-tmpl-3 {
     lr = 0.0003
 }
 
+s2s {
+    d_hid_dec = 100
+    n_layers_dec = 1
+    attention = "Bahdanau". // Options are 'Bahdanau', 'bilinear' or 'none'.
```

Comment:
Can you give a quick 1-line description of the differences between Bahdanau and bilinear ?

Reply:
Also nit, remove the "."